### PR TITLE
Create ridge regression path figures

### PR DESCRIPTION
**Pull request by karalleyna**
I have created two figures showing Ridge Regression path. 

Ekran Resmi 2021-04-21 23 53 05
Ekran Resmi 2021-04-21 23 53 11

I decided to use larger alpha values after reading the related part of the book in order for figures to be consistent those in the book. So, it can be changed according to your feedback. @mjsML

### CODE REVIEW

**Review by murphyk — CHANGES_REQUESTED** [review]
Looks good, but please make a few changes
- just use numpy, no need to use jax in this case
- it's confusing to have alphas and alphas_cv, and you end up fitting the model more times than necessary. Please just use alphas_cv
- The dof on the x-axis for the first plot is different from the matlab code. I don't understand your equation: where does sum_d var(x_d)/(var(x_d) + alpha) come from?
- Best to just plot alpha on the x-axis rather than dof or integer index

**Comment by murphyk** [comment]
Also, please link to the issue that you are solving, by saying 
closes #204

**Comment by karalleyna** [comment]
The former figure in the code, the regression path was plotted like a figure whose x axis showed dof which I supposed it means the degrees of freedom of ridge regression, whose formula is also given in the book. Therefore, I plotted two figures, one is showing the weights against dof and the latter one was plotted via the transpose of the coefficient matrix since I tried to be consistent to the previous code as well as the figure in the book. Furthermore, since there are issues about the use of jax.numpy in colab, so I used jax.numpy even if the use of numpy is mentioned within the guidelines.

**Comment by murphyk** [comment]
Where did you see the equation for dof in the book? 
Also, 
It seems that #410 also implements this ridge path code. Since their code is simpler, I will close this issue.

**Comment by karalleyna** [comment]
I have found it at Chapter 11.3.2.
